README.md

```diff
@@ -413,7 +413,7 @@ pip install --upgrade "jax[cpu]"
 ```
 
 On Linux, it is often necessary to first update `pip` to a version that supports
-`manylinux2014` wheels.
+`manylinux2014` wheels. Note these existing wheels are currently for `x86_64` architectures only. Trying to pip install with other architectures may lead to `jaxlib` not being installed alongside `jax`, although `jax` may successfully install.
```

Comment:
The wording here is a bit misleading, because we do build and release Mac ARM wheels. So you should probably put "Linux" somewhere. Perhaps reword this change as:
"Note that on Linux, we release wheels for the `x86_64` architecture only at the moment. Other architectures require building from source."

Reply:
Good point, I reworded it to emphasize that caveat applies to Linux architectures only.